Dockerfile

```diff
@@ -101,64 +266,69 @@ FROM scratch as ecs-resource-agent
 COPY --from=build /etc/ssl /etc/ssl
 COPY --from=build /etc/pki /etc/pki
 # Copy binary
-COPY --from=build /src/bottlerocket-agents/bin/ecs-resource-agent ./
+COPY --from=build-src /src/bottlerocket-agents/bin/ecs-resource-agent ./
+# Copy licenses
+COPY --from=build-src /usr/share/licenses/testsys /licenses/testsys
 
 ENTRYPOINT ["./ecs-resource-agent"]
 
 # =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
 # Builds the ECS test agent image
-FROM public.ecr.aws/amazonlinux/amazonlinux:2 as ecs-test-agent
+FROM scratch as ecs-test-agent
 # Copy binary
-COPY --from=build /src/bottlerocket-agents/bin/ecs-test-agent ./
+COPY --from=build-src /src/bottlerocket-agents/bin/ecs-test-agent ./
+# Copy licenses
+COPY --from=build-src /usr/share/licenses/testsys /licenses/testsys
 
 ENTRYPOINT ["./ecs-test-agent"]
 
 # =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
 # Builds the Sonobuoy test agent image
 FROM public.ecr.aws/amazonlinux/amazonlinux:2 AS sonobuoy-test-agent
-ARG GOARCH
 ARG ARCH
-ARG SONOBUOY_VERSION=0.53.2
-ARG SONOBUOY_URL=https://github.com/vmware-tanzu/sonobuoy/releases/download/v${SONOBUOY_VERSION}/sonobuoy_${SONOBUOY_VERSION}_linux_${GOARCH}.tar.gz
-ARG AWS_IAM_AUTHENTICATOR_URL=https://amazon-eks.s3.us-west-2.amazonaws.com/1.21.2/2021-07-05/bin/linux/${GOARCH}/aws-iam-authenticator
+# TODO remove unzip once aws-cli moves out
+RUN yum install -y unzip iproute && yum clean all
 ARG AWS_CLI_URL=https://awscli.amazonaws.com/awscli-exe-linux-${ARCH}.zip
-RUN yum install -y tar gzip unzip iproute && yum clean all
 
-# Download aws-iam-authenticator
-RUN temp_dir="$(mktemp -d --suffix aws-iam-authenticator)" && \
-    curl -fsSL "${AWS_IAM_AUTHENTICATOR_URL}" -o "${temp_dir}/${AWS_IAM_AUTHENTICATOR_URL##*/}" && \
-    chmod 0755 "${temp_dir}/${AWS_IAM_AUTHENTICATOR_URL##*/}" && \
-    mv "${temp_dir}/${AWS_IAM_AUTHENTICATOR_URL##*/}" /usr/bin/aws-iam-authenticator && \
-    rm -rf ${temp_dir}
+# Copy aws-iam-authenticator binary
+COPY --from=aws-iam-authenticator-build /tmp/aws-iam-authenticator /usr/bin/aws-iam-authenticator
+# Copy aws-iam-authenticator licenses
+COPY --from=aws-iam-authenticator-build /usr/share/licenses/aws-iam-authenticator /licenses/aws-iam-authenticator
 
+# TODO move this out, get hashes, and attribute licenses
```

Comment:
Commenting here for easy issue creation. aws-cli may need the same treatment as the other libraries with licenses.

Reply:
#286